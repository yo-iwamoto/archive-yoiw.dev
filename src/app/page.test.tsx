import Page from './page';
import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';

describe('IndexPage', () => {
  it('renders unchanged', () => {
    const { getByRole, asFragment } = render(<Page />);

    expect(getByRole('heading', { name: 'Hello world', level: 1 })).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
