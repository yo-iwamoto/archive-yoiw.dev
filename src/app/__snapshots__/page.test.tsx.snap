// Vitest Snapshot v1

exports[`IndexPage > renders unchanged 1`] = `
<DocumentFragment>
  <h1>
    Hello world
  </h1>
</DocumentFragment>
`;
